src/main/groovy/com/bmuschko/gradle/clover/AggregateDatabasesTask.groovy

```diff
@@ -34,6 +41,7 @@ class AggregateDatabasesTask extends DefaultTask {
 
         if(existsAtLeastOneCloverDbFile(cloverDbFiles)) {
             ant.taskdef(resource: 'cloverlib.xml', classpath: getCloverClasspath().asPath)
+            ant.property(name: 'clover.license.path', value: getLicenseFile().canonicalPath)
```

Comment:
This change was needed because for some reason my test task was up-to-date, but the cloverAggregateDatabases task was not.  When this ran, it failed because there was no license file.  I wanted to write an integration test here, but, I could not figure out how to simulate this case in test.  If you have any suggestions, I'd be happy to implement the test.